database/migrations/2025_03_31_124158_support_model_name_on_site.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // Add position column on measurement table.
+        echo "Adding 'processormodelname' column on 'siteinformation' table" . PHP_EOL;
+        Schema::table('siteinformation', function (Blueprint $table) {
+            $table->text('processormodelname')->default(0);
```

Comment:
This column should be nullable, with no default value.  That way, we can differentiate between no data and missing data, just like the other columns.